jscomp/others/belt_List.ml

```diff
@@ -692,6 +692,13 @@ let setAssocU xs x k eq =
 
 let setAssoc xs x k eq = setAssocU xs x k (fun [@bs] a b -> eq a b)
 
+let sort xs cmp =
+  let arr = ref (toArray xs) in
```

Comment:
why a ref needed here

Reply:
sortU/sort

Reply:
Ok, I'll add a sortU version. I think I was confused about whether I could use let and it still be mutated by the in-place sort. It works fine without the ref.